app/api_clients/control_panel_api.js

```diff
@@ -11,10 +11,55 @@ class ControlPanelAPIClient extends APIClient {
       header: `JWT ${user.id_token || 'invalid token'}`,
     };
   }
+
+  request(endpoint, { method = 'GET', body = null, params = {} } = {}) {
+    return super.request(endpoint, { method, body, params })
+      .catch((error) => {
+        if (DjangoError.match(error)) {
+          throw new DjangoError(error);
+        }
+        throw error;
+      });
+  }
 }
 
 exports.APIClient = ControlPanelAPIClient;
 
 const api = new ControlPanelAPIClient(config.api);
 
 exports.api = api;
+
+
+class DjangoError extends APIError {
```

Comment:
I wonder if we could keep all these errors classes in a `app/api_clients/errors.js`.

Reply:
The DjangoError is pretty specific to the control_panel_api, so I think it belongs with it